pages/how-to-contribute.md

```diff
@@ -8,6 +8,7 @@ Contributions are welcome to all Apache Accumulo repositories. While most contri
 there are other ways to contribute to Accumulo:
 
 * communicate on one of the [mailing lists](/contact-us/#mailing-lists)
+* participate in design discussions on [Confluence][confluence]
```

Comment:
I would not characterize Confluence as a place for "discussions". The mailing list is for that. Confluence is for drafting/posting/commenting-on design docs.

I suggest:

```suggestion
* view/draft/comment on designs or other docs in [Confluence][confluence]
```